inst/WORDLIST

```diff
@@ -26,28 +26,32 @@ Bazett's
 BDS
 Biologics
 BMI
-cdisc
 CDISC
 Changelog
 censorings
 codebase
-Codebase
 CRF
+correctitude
 CQ
+cyclomatic
 datepart
 datetime
 developers’
+ds
+dtc
+DTC
 DuBois
-Durations
+durations
 EMA
 FACM
+familiarise
+familiarised
 Fridericia
 Fridericia's
 Fujimoto
 functions’
 funder
 Gehan
-github
 Github
```

Comment:
I think GitHub is the one we want.  So we want Github to be flagged as inccorect 
![image](https://user-images.githubusercontent.com/10111024/154859605-699afcdf-1bde-485c-b0c0-7928ec292ae9.png)

Reply:
Yes, let's always use the correct spelling "GitHub".